packages/curriculum-parser/plugins/question/answers.js

```diff
@@ -12,16 +12,14 @@ module.exports = function questionAnswers () {
   }
 
   function parseQuestionAnswersList (temp) {
-    return function one(node, index, parent) {
-      if (node.type === 'list'
-        && !node.ordered
-        && index === parent.children.length - 1
-        && (
-          parent.type === 'root' ||
-          (parent.type === 'section' && parent.question)
-        )
+    return function one (node, index, parent) {
+      if (
+        node.type === 'list' &&
+        !node.ordered &&
+        index === parent.children.length - 1 &&
+        (parent.type === 'root' ||
+          (parent.type === 'section' && parent.question))
```

Comment:
I wonder why the formatting changed here

Reply:
oh it's my fault
I have globally installed javascript standard + auto-prettier format on a keybind
want me to revert?

Reply:
It's no biggie in my opinion. We should probably have a formatter to handle this like in `enki-common` of `monoenki`.

Reply:
well to get to that we just need a shared `.eslintrc` file. the formatter will follow those rules

javascript standard is just like eslint with the config: `extends: 'standard'`